oaknational/Oak-Web-Application
# Feat/lesson list item numbered

## Description

- Adds numbers to lesson list items

## Issue(s)

Fixes #1419 

## How to test

1. Go to lesson list page and you should see numbers, search lesson list items should have no numbers
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
